pages/resources/quests.mdx

```diff
@@ -0,0 +1,518 @@
+  "quest_id": "alien-quest",
+  "enrolled_at": "2077-01-01T11:59:59+00:00",
+  "completed_at": null,
+  "claimed_at": null,
+  "claimed_tier": null,
+  "last_stream_heartbeat_at": null,
+  "stream_progress_seconds": 0,
+  "dismissed_quest_content": 0,
+  "progress": {}
+}
+```
+
+##### Example Response
+
+<RouteHeader method="POST" url="/quests/{quest.id}/claim-reward">
+  Claim A Quest's Reward
+</RouteHeader>
+
+Claims the quest's reward and sets both the `completed_at` and `claimed_at` fields of the [User Status Object](#quest-user-status-object) to the current timestamp.
+
+Note that this endpoint _requires_ a valid captcha key.
+
+##### Example Response (Avatar Decorations)
+
+```json
+{
+  "claimed_at": "2024-04-17T23:30:41.000321+00:00",
+  "entitlements": [
+    {
+      "id": "1230299425620885624",
+      "sku_id": "1226939756617793606",
+      "application_id": "1096190356233670716",
+      "user_id": "222069018507345921",
+      "deleted": false,
+      "type": 10,
+      "tenant_metadata": {},
+      "gift_code_flags": 0,
+      "promotion_id": null
+    }
+  ],
+  "errors": []
+}
+```
+
+<RouteHeader method="POST" url="/quests/{quest.id}/reward-code">
+  Fetch The Received Reward Code
+</RouteHeader>
+
+Retrieves the reward code for the specified [platform.](#quest-platform-type)
+
+###### JSON Params
+
+| Field    | Type                                       | Description                             |
+| -------- | ------------------------------------------ | --------------------------------------- |
+| location | integer                                    | Used for analytics (probably)           |
+| platform | [platform type enum](#quest-platform-type) | Which platform to retrieve the code for |
+
+#### Example Response
+
+```json
+{
+  "quest_id": "alien-quest",
+  "code": "dolfies-pls-give-me-the-alien-tech-🥺",
+  "platform": 0,
+  "user_id": "222069018507345921",
+  "claimed_at": "2077-01-01T18:41:29.706194+00:00",
+  "tier": null
+}
+```
+
+<RouteHeader method="POST" url="/quests/{quest.id}/reward-code">
+  Send Stream Heartbeat
+</RouteHeader>
+
+Tells the server to update the `last_stream_heartbeat` and `stream_progress_seconds` fields, and is used for keeping track of how long the stream has been running for and for checking if the user has met the stream duration requirement.
+
+Returns a [quest user status object.](#quest-user-status-object)
+
+###### JSON Params
+
+| Field      | Type   | Description                                                                                                                                                   |
+| ---------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| stream_key | string | A string that is a combination of the guild id, the voice channel id and the user's id. ex. `guild:169256939211980800:1050497861969793164:222069018507345921` |
```

Comment:
```suggestion
| stream_key | string | The encoded key of the stream (e.g. `guild:169256939211980800:1050497861969793164:222069018507345921`) |
```
An example is fine but this shouldn't be doc'd here as it's common throughout the streams API